internal/cmdrender/executor.go

```diff
@@ -33,13 +36,17 @@ import (
 // Executor hydrates a given pkg.
 type Executor struct {
 	PkgPath string
+	Printer printer.Printer
 }
 
 // Execute runs a pipeline.
-func (e *Executor) Execute() error {
+func (e *Executor) Execute(ctx context.Context) error {
+	op := errors.Op("fn.render")
```

Comment:
So I think we should standardize on the pattern used in upspin, which is to create these as constants: 
```
const op errors.Op = "fn.render"
```

I feel this makes it stand out from the rest of the code in the function and makes it clear that this is a constant.

Reply:
Good point. Done.